monetio/models/raqms.py

```diff
@@ -30,17 +32,37 @@ def open_dataset(fname, *, surf_only=False):
     ds = xr.open_dataset(names[0], drop_variables=["theta"])
     ds = _fix(ds, surf_only=surf_only)
 
+    # convert all gas species to ppbv
```

Comment:
You could move this to `_fix` or a `_fix_units` called by `_fix()` to you don't have to duplicate it?